src/vs/workbench/contrib/terminal/browser/terminalTabbedView.ts

```diff
@@ -214,6 +230,44 @@ export class TerminalTabbedView extends Disposable {
 		this._chatEntry?.update();
 	}
 
+	private _ensureResizeOverlay(): HTMLElement {
+		if (!this._resizeOverlay) {
+			this._resizeOverlay = $('.terminal-resize-overlay');
+			this._resizeOverlay.setAttribute('role', 'status');
+			this._resizeOverlay.setAttribute('aria-live', 'polite');
+			this._parentElement.append(this._resizeOverlay);
+			this._register(toDisposable(() => this._resizeOverlay?.remove()));
+		}
+		return this._resizeOverlay;
+	}
```

Comment:
Playing around with this more, it should probably actually live in `TerminalInstance` since this falls over right now when there are multiple terminals or tabs are shown:

<img width="2030" height="778" alt="Image" src="https://github.com/user-attachments/assets/ed7dbeb3-e955-49f0-bcb6-4922dfa31bf2" />

It's also fine that it shows for all split terminals at once, not just the active one. But we do want to make sure that the window startup and terminal start up problem is handled. Moving it into `TerminalInstance` should make it a lot easier to accomplish this as it knows when it was just launched, it knows the first resize event, etc.